src/trips_ms/src/main/java/com/cringe/travels/trips/trip/TripService.java

```diff
@@ -0,0 +1,163 @@
+    public void updateHotelsRooms(JSONObject jsonObject) {
+
+        JSONArray listIds = jsonObject.getJSONArray("trip_offers_id");
+        String operationType = jsonObject.getString("operation_type");
+        String roomType = jsonObject.getString("room_type");
+
+        for (int i = 0; i < listIds.length(); i++) {
+            String id = listIds.getString(i);
+            Optional<Trip> tripOpt = repository.findById(id);
+            if (tripOpt.isPresent()) {
+                Trip trip = tripOpt.get();
+                int freeSeats = trip.getHotel().getRooms().get(roomType).getAvailable();
+                if (operationType.equals("add")) {
+                    freeSeats++;
+                } else {
+                    freeSeats--;
+                }
+                trip.getHotel().getRooms().get(roomType).setAvailable(freeSeats);
+                repository.save(trip);
+                logger.info("UPDATED HOTEL ROOMS IN TRIP: " + trip.getId());
+            }
+        }
+    }
+
+    public void updateTripStatus(JSONObject jsonObject) {
+
+        JSONArray listIds = jsonObject.getJSONArray("trip_offers_id");
+        boolean isHotelBookedUp = jsonObject.getBoolean("is_hotel_booked_up");
+
+        for (int i = 0; i < listIds.length(); i++) {
+            String id = listIds.getString(i);
+            Optional<Trip> tripOpt = repository.findById(id);
+            if (tripOpt.isPresent()) {
+                Trip trip = tripOpt.get();
+                trip.setBookedUp(isHotelBookedUp);
+                repository.save(trip);
+                logger.info("UPDATED TRIP STATUS IN TRIP: " + trip.getId());
+
+            }
+        }
+    }
+
+    public void updateTransport(JSONObject jsonObject) {
+
+        JSONArray listIds = jsonObject.getJSONArray("trip_offers_id");
+        String connectionId = jsonObject.getString("connection_id");
+        String operationType = jsonObject.getString("operation_type");
+        int headCount = jsonObject.getInt("head_count");
+
+
+        for (int i = 0; i < listIds.length(); i++) {
+            String id = listIds.getString(i);
+            Optional<Trip> tripOpt = repository.findById(id);
+            if (tripOpt.isPresent()) {
+                Trip trip = tripOpt.get();
+                trip.getFrom().forEach((key, transport) -> {
+                    try {
+                        if (transport.getPlane().getId().equals(connectionId)) {
+                            int seatsLeft = transport.getPlane().getSeatsLeft();
+                            if (operationType.equals("add")) {
+                                seatsLeft = seatsLeft + headCount;
+                            } else {
+                                seatsLeft = seatsLeft - headCount;
+                            }
+                            transport.getPlane().setSeatsLeft(seatsLeft);
+                            repository.save(trip);
+                            logger.info("UPDATED TRANSPORT IN TRIP: " + trip.getId());
```

Comment:
Dodaj liczbę miejsc zaktualizowanych, bo to co jest teraz to w logach to będzie za mało